Projects/Server/MultiGameRunner.cpp

```diff
@@ -0,0 +1,42 @@
+#include "MultiGameRunner.h"
+
+MultiGameRunner::MultiGameRunner(const AbstractPlayer::Ptr& player0, const AbstractPlayer::Ptr& player1, const size_t& totalGamesToBePlayed = 1)
+	: _player0{ player0 }, _player1{ player1 }, _totalGamesToBePlayed{ totalGamesToBePlayed }
+{
+	_gameOutcomes[State::Outcome::P0WIN];
+	_gameOutcomes[State::Outcome::P1WIN];
+	_gameOutcomes[State::Outcome::DRAW];
+	_currentGameCount = 0;
+}
+
+
+void MultiGameRunner::play() 
+{
+	GameHandler gameHandler{};
+
+	for (size_t i{ 0 }; i < _totalGamesToBePlayed; ++i) 
+	{	
+		gameHandler.init(_player0, _player1);
+		
+
+		State::Outcome outcome = gameHandler.play();
+		
+		++_currentGameCount;
+		++_gameOutcomes[outcome];
+		
+		std::cout << *this;
+	}
+}
+
+std::ostream& operator<<(std::ostream& os, const MultiGameRunner& multiGameRunner) 
+{	
+	os << "Games played: " << multiGameRunner._currentGameCount << " / " << multiGameRunner._totalGamesToBePlayed << "\n";
```

Comment:
What's the point of having `_currentGameCount` here?

Reply:
It's removed